src/main/kotlin/bio/messages/MessageCreationRequest.kt

```diff
@@ -0,0 +1,5 @@
+package bio.messages
+
+data class MessageCreationRequest(
```

Comment:
`MessageCreationRequest` imo should have the `channelId` specified in this data class OR specified in routing, like  `POST /channels/{channel_id}/messages`